crawler/src/main/java/nl/fhict/happynews/crawler/CrawlerController.java

```diff
@@ -45,6 +46,34 @@ public CrawlerController() {
         logger = LoggerFactory.getLogger(CrawlerController.class);
     }
 
+
```

Comment:
I meant the method insertSources, should have tagged the method line instead of this one